solutions/kotlin/01-dr6/code/src/main/kotlin/Main.kt

```diff
@@ -15,7 +16,7 @@ fun main(args: Array<String>) {
         databaseFile.skip(16) // Skip the first 16 bytes of the header
         val pageSizeBytes = ByteArray(2) // The following 2 bytes are the page size
         databaseFile.read(pageSizeBytes)
-        val pageSize = java.nio.ByteBuffer.wrap(pageSizeBytes).short.toInt()
+        val pageSize = ByteBuffer.wrap(pageSizeBytes).short.toUShort()
```

Comment:
@andy1li this actually looks even cleaner. Can we do this for the Java one too? Or does it not support `toUShort`?